node12/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:12-stretch
+
+RUN apt-get update && \
+    apt-get install -y chromium=70.0.3538.110-1~deb9u1 firefox-esr xvfb libxi6 libgconf-2-4 default-jre && \
```

Comment:
I would also use that latest version of chrome

Reply:
only in the new node12 img, or can we also update it for node10? maybe we should release before merging that as a safeguard.